src/unity/python/turicreate/toolkits/_mxnet_to_coreml/_layers.py

```diff
@@ -81,6 +81,9 @@ def convert_reshape(net, node, module, builder):
     name = node['name']
     param = _get_attr(node)
     target_shape = literal_eval(param['shape'])
+    if param['shape'] == '(0, -1)':
```

Comment:
Could you explain what this logic does?

Reply:
For flatten layers, the mxnet converter calls `convert_reshape` (which is fine because flatten is a kind of reshape) but it doesn't actually call `convert_flatten` and instead throws a `NotImplemented` Error since the shape parameters for a flatten layer when `convert_reshape` is called are `(0,-1)`. @gustavla looked at this code during our code review and recommended that I change this to not do a string comparison but to do a value comparison for the tuple, i.e. target_shape. Will make that change.